src/routes/index.js

```diff
@@ -2,16 +2,26 @@ const express = require('express');
 const router = express.Router();
 const Products = require('../models/Products');
 
-router.get('/', async (req,res) => {
+router.get('/', async (req, res) => {
     try {
-        const prds = await Products.find();
+        const products = await Products.find();
+        if (!products) return res.status(404).render("pages/error", {
+            title: "Moramia <3",
```

Comment:
404 title should be different. Try something like: `Page not found`

Reply:
Done